src/components/signup/Signin.jsx

```diff
@@ -1,30 +1,65 @@
 import { useState } from "react";
 import { useNavigate } from "react-router-dom";
+import { useLogin } from "../../contexts/login-context/login-context";
 import "./signup.css";
-export default function Signin() {
-  //   const { state: userState, dispatch: userDispatch } = useLogin();
-  //   const { email, password } = userState.user;
-  const [showPassword, setShowPassword] = useState(false);
+import axios from "axios";
+
+export default function Signin({ setIsSignUp, setShowAlert, setAlertMsg }) {
+  const { dispatchUser } = useLogin();
   const navigate = useNavigate();
+  const [showPassword, setShowPassword] = useState(false);
+  const [loginForm, setLoginForm] = useState({
+    email: "",
+    password: "",
+  });
+
   const changeHandler = event => {
     event.preventDefault();
-    // return userDispatch({ type: "USER_FORM", payload: event.target });
+    setLoginForm(() => ({
+      ...loginForm,
+      [event.target.name]: event.target.value,
+    }));
   };
 
   const toggleShowPassword = e => {
     e.preventDefault();
-    console.log("in toggle pwd");
```

Comment:
remove console.log when you push in production

Reply:
this is already a removed code line